service-api/app/test/AppTest/Service/ActorCodes/ActorCodeServiceTest.php

```diff
@@ -155,7 +155,7 @@ function (string $id) {
             ),
             Argument::exact('test-user'),
             Argument::exact('test-uid'),
-            Argument::exact(1)
+            Argument::exact("1")
```

Comment:
If this is a string then it should be single quoted to stop the interpolation overhead. That said this line could also be 
```suggestion
            '1'
```